.github/workflows/waspc-ci.yaml

```diff
@@ -188,22 +188,22 @@ jobs:
 
       - name: Headless - Cache Playwright Browsers for Playwright's Version
         id: headless-cache-playwright-browsers
-        if: matrix.os == 'ubuntu-20.04'
+        if: matrix.os == 'ubuntu-latest'
         uses: actions/cache@v3
         with:
           path: ~/.cache/ms-playwright
           key: playwright-browsers-${{ env.PLAYWRIGHT_VERSION }}
 
       - name: Headless - Setup Playwright
         id: headless-setup-playwright
-        if: steps.cache-playwright-browsers.outputs.cache-hit != 'true' && matrix.os == 'ubuntu-20.04'
+        if: steps.cache-playwright-browsers.outputs.cache-hit != 'true' && matrix.os == 'ubuntu-latest'
         run: |
           cd examples/todoApp
           npx playwright install --with-deps
 
       - name: Headless - Run Playwright Tests
         id: headless-run-playwright-tests
-        if: matrix.os == 'ubuntu-20.04'
+        if: matrix.os == 'ubuntu-latest'
```

Comment:
Wow we are really repeating this ubuntu version in a ton of places. I believe there is a way to have a variable here in Github Action YAML, maybe this is a good moment to extract it? If not, ok we can do it later, I won't push too much, but am tempted.

Reply:
> I believe there is a way to have a variable here in Github Action YAML

Unfortunately no - you can have something like a variable (YAML anchor) but you can't use it in strings and conditional clauses. To do that, I'd have to define an env variable and change the syntax of `if` clauses to check the env, which is a bit too much for my taste.